test/rules.js

```diff
@@ -12,3 +12,35 @@ test('badge - incorrect source', async t => {
 	t.is(result.ruleId, 'awesome-badge');
 	t.is(result.message, 'Incorrect badge source');
 });
+
+test('list-item - incorrect item prefix', async t => {
+	const results = (await m({filename: 'fixtures/list-item/1.md'})).messages;
+	for (const result of results) {
```

Comment:
This is a very fragile way to test it. There might at some point be other violations in the fixtures, so the messages array might also contain other violations. Better to use Array.filter to get the `awesome-list-item` rule and then check the message.